src/plays/github-user-finder/components/ProfileCard.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useResultContext } from '../context/ResultContext';
+import formatDate from '../utilities/formateDate';
+
+const ProfileCard = () => {
+  const { results } = useResultContext();
+
+  return (
+    <div className="card m-auto p-8">
+      <img alt="User Image" className="avatar flex self-center" src={results.avatar_url} />
```

Comment:
When the app loads first the image is not available, hence a broken image

![image](https://user-images.githubusercontent.com/3633137/216959895-bbce511c-23e9-4040-9657-37f5e9e5d00b.png)

You can put a condition to handle it.

Reply:
Ok sure